nim-compile.el

```diff
@@ -133,19 +129,22 @@ The config file would one of those: config.nims, PROJECT.nim.cfg, or nim.cfg."
     (remove-hook 'compilation-filter-hook 'nim--colorize-compilation-buffer t)))
 
 ;;;###autoload
-(defun nim-compile ()
-  "Compile and execute the current buffer as a nim file.  All output is writton into the *compilation* buffer."
-  (interactive)
+(defun nim-compile (command)
+  "Compile and execute the current buffer as a nim file.  All output is writton into the *compilation* buffer.  COMMAND will be used as the compiler command."
+  (interactive
+   (list
+    (let ((command (eval compile-command)))
```

Comment:
well I don't know if I need it, because that part is copy paste code from the interactive section from ``compile`` in ``compile.el``, so that the behavior is identical.